src/main/java/de/tum/in/www1/artemis/web/rest/ProgrammingExerciseResource.java

```diff
@@ -734,6 +733,12 @@ public ProgrammingExerciseResource(ProgrammingExerciseRepository programmingExer
 
         public static final String SETUP = PROGRAMMING_EXERCISES + "/setup";
 
+        public static final String NO_LOCAL_SETUP = PROGRAMMING_EXERCISES + "/no-local-setup";
```

Comment:
please move these constants to another file and do not change ProgrammingExerciseResource at all

Reply:
i moved the endpoints to the corresponding simulation classes